### PR TITLE
Display all collection actions after edit

### LINKED ISSUE #743: Incomplete list of collection actions available after collection editing
With Admin permissions, when I first look at a collection I get these actions.

screen shot 2015-03-17 at 1 23 10 pm

After editing and saving the collection, I get a reduced set.

screen shot 2015-03-17 at 1 23 25 pm

### PR DESCRIPTION
**Pull request by mgrauer**
Fixes #743.

~~I didn't have enough time to add a client side test for this.~~

### CODE REVIEW

**Comment by cpatrick** [comment]
Open an issue to add a test for this and it LGTM.

File: clients/web/test/spec/collectionSpec.js [hunk 1]
```diff
@@ -58,8 +58,79 @@ describe('Test collection actions', function () {
...
+            return girder.numberOutstandingRestRequests() === 0;
+        }, 'dialog rest requests to finish');
+
+        waitsFor(function () {
+            return $('.g-collection-description').text().match('New Description');
+        }, 'description to be updated to new text');
+
+        waitsFor(function () {
+            return $('.g-collection-actions-button:visible').is(':enabled');
+        }, 'collection actions link to appear');
```
Comment (zachmullen) [inline clients/web/test/spec/collectionSpec.js]: In theory multiple consecutive `waitsFor` blocks could be combined into one.

Comment (zachmullen) [inline clients/web/test/spec/collectionSpec.js]: That said, if you think it's cleaner this way, LGTM.

**Comment by mgrauer** [comment]
I think it is easier to see the intent this way, rather than combining.

**Pull request merged**